app/services/group/v1x1_seed.rb

```diff
@@ -0,0 +1,74 @@
+module Group
+  class V1x1Seed
+    attr_reader :status
+    CATALOG_ADMINISTRATOR_GROUP = "Catalog Administrators".freeze
+
+    def initialize
+      @request = Insights::API::Common::Request.current
+      raise Catalog::NotAuthorized unless @request.user.org_admin?
+    end
+
+    def tenant
+      @request.tenant
+    end
+
+    def code(status)
+      @status = status
+    end
+
+    def process
+      lookup_groups
+      mark_as_seeded
+
+      if @seeded.data.present?
```

Comment:
We are not going to run if any groups exist, correct?  With the current state of the RBAC service being used by many services I do not think we should gate on this anymore.  Maybe check for the existence of the CATALOG_ADMINISTRATOR_GROUP?

Or we do not gate this action at all.  Re-running seeding should not cause any issue and the result would just be adding the current org_admin as a Catalog Admin.